Displays/PIXIE.txt

```diff
@@ -0,0 +1,117 @@
+PIXIE
+
+
+DCPU-16 Hardware Info:
+    Name: PIXIE - Compatible Video Generator
+    ID: 0x774df615, version: 0x1802
+    Manufacturer: 0x00000000 (Manufacturer not yet found)
```

Comment:
I'd recommend going ahead and making up a value for Agnus Micro Devices or whatever and put that in here